students/11th/choonghee-lee/account/models.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+
+class User(models.Model):
+    username = models.CharField(max_length = 30, blank = True, unique = True)
```

Comment:
User Table 내에 이름으로 username 보다는 name 으로 해도 User Table에 있기 때문에 user name으로 인식할 수 있을 것 같습니다.
```suggestion
    name = models.CharField(max_length = 30, blank = True, unique = True)
```